src/libraries/System.Globalization/tests/System/Globalization/RegionInfoTests.cs

```diff
@@ -63,6 +66,21 @@ public void CurrentRegion()
             }
         }
 
+        [ConditionalFact(nameof(RegionInfoPropertyTests.IcuIsNotLoaded))]
```

Comment:
Is this really a ICU-specific test, or is more just a Windows-specific test?

Do you expect that the ICU option that @safern is working on is going to affect RegionInfo?

Reply:
We have to do more changes for initializing CurrentCultures and Regions on Windows when using ICU. This need to be done in other PR's as part of @safern next steps.

Reply:
Should the condition be just for Windows for now, and not talk about the ICU at all?

What you are saying, we will need to touch this code for ICU work anyway, so the future proofing does not help much.

Reply:
I didn't use IsWindows because when Santi enables the first step we'll start failing as we'll be running on Windows and not doing the right initialization yet.

Reply:
Right, but I expect that we will have 100 other places with the same problem. All of them will need to be fixed, the "IsIcu" property will need to be moved into more central place, etc. So the code in this file will need to be redone anyway.

Reply:
Ok, I'll change the condition to check Windows then as I am not feeling strongley about that either way.